harness/determined/common/requests.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, server_hostname: Optional[str]) -> None:
         self.mount("https://", HTTPAdapter(server_hostname))
 
 
-def request(method: str, url: str, **kwargs: Any) -> requests.Response:
+def request(method: str, url: str, **kwargs: Any) -> Any:
```

Comment:
types-requests source code suggests it still should be a response: https://github.com/python/typeshed/blob/master/stubs/requests/requests/sessions.pyi#L107
is this mypy misbehaving again?..

Reply:
hm... not sure.  I'll just fix it with a type-cast.